package/gluon-status-page-api/src/stations.c

```diff
@@ -58,8 +62,46 @@ bool interface_is_valid(const char *ifname) {
   return access(path, F_OK) == 0;
 }
 
+int interface_get_lowest(const char *ifname, char *hwifname);
+int interface_get_lowest(const char *ifname, char *hwifname) {
```

Comment:
right, it should be static. the prototype is needed as I'm calling interface_get_lowest() recursively.